client/.babelrc

```diff
@@ -1,15 +1,27 @@
 {
-  "presets": ["flow", "es2015", "stage-2", "react"],
+  "presets": [
+    "flow",
+    "es2015",
+    "stage-2",
+    "react"
+  ],
   "env": {
-    "test": {
+    "development": {
       "plugins": [
-        "transform-es2015-modules-commonjs"
+        "flow-react-proptypes",
+        [
+          "import",
+          {
+            "libraryName": "antd",
```

Comment:
Are we going to be using Ant for all of our components now? Just wondering what the motivation is here.

Reply:
Ah I see in our #react channel. So are we just using Ant for its grid system? If so, is there a way to just import that instead of the whole framework?

Reply:
@julianguyen You will be pleased to know that using [babel-plugin-import](https://github.com/ant-design/babel-plugin-import) (which is maintained by the ant team well) will load only the components you imported into your code, not the whole framework.

I think to answer the question of whether all of our components will use Ant, I say if any of the volunteers want to override the style of an Ant component, they have the option to now. And it won't unnecessarily bloat the code, so we have the flexibility to incrementally bring in Ant components as needed.